mne/viz/circle.py

```diff
@@ -424,3 +425,77 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
 
     plt_show(show)
     return fig, axes
+
+
+def plot_channel_labels_circle(labels, colors=None, picks=None, **kwargs):
+    """Plot labels for each channel in a circle plot.
+
+    Parameters
+    ----------
+    labels : dict
+        Lists of labels (values) associated with each channel (keys).
+    colors : dict
+        The color (value) for each label (key).
+    picks : list | tuple
+        The channels to consider.
+    **kwargs : kwargs
```

Comment:
I prefer to avoid kwargs if possible.

Reply:
There are 24 arguments that could possibly be used from `mne.viz.plot_connectivity_circle` to change the plotting style so I'm not sure this is possible. It especially provides the ability to change fontsizes and colors which is really nice for plots.

Reply:
Hum… let’s say I am getting old and more flexible… I can live with that